### PR TITLE
update chart release action to use CR release v1.4.0

### PR DESCRIPTION
**Pull request by cpanato**
- update chart release action to use CR release v1.4.0

### CODE REVIEW

File: .github/workflows/test-action.yml
Comment (unguiculus) [inline .github/workflows/test-action.yml:66]: You could use the `--exit-code` flag. The else branch is unnecessary.
```suggestion
        if ! git diff --stat --exit-code; then
          echo 'should be clean'
          exit 1
        fi
```

File: action.yml
Comment (unguiculus) [inline action.yml:53]: Always quote variables
```suggestion
          echo $install >> "$GITHUB_PATH"
```

File: action.yml
Comment (unguiculus) [inline action.yml:56]:
```suggestion
          echo ${{ inputs.install_dir }} >> "$GITHUB_PATH"
```

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Review by scottrigby — COMMENTED** [review]
Apart from @unguiculus suggestions, this lgtm! Once we merge this and release a new version of the action, we can update https://github.com/helm/charts-repo-actions-demo 😊

IF we want to use helm/charts-repo-actions-demo repo to test this before merging – which we have done in the past – we can do by pointing this PRs branch on your fork and ensure all is well there. Not sure if we need that anymore but it has been helpful in the past

File: action.yml
Comment (cpanato) [reply action.yml:56]: thanks for your review @unguiculus

**Review by unguiculus — APPROVED** [review]
LGTM

**Pull request merged**
